azurerm/internal/services/network/tests/virtual_network_gateway_connection_resource_test.go

```diff
@@ -113,6 +113,27 @@ func TestAccAzureRMVirtualNetworkGatewayConnection_ipsecpolicy(t *testing.T) {
 	})
 }
 
+func TestAccAzureRMVirtualNetworkGatewayConnection_trafficselectorpolicy(t *testing.T) {
```

Comment:
_minor_ but could we update the test name to match the other resources?

```suggestion
func TestAccAzureRMVirtualNetworkGatewayConnection_trafficSelectorPolicy(t *testing.T) {
```